Cannot initialize module SuppressionFilter Error

The maven module mnemonic-nvml-vmem-service shows the following errors.
```
[INFO] --- maven-checkstyle-plugin:3.0.0:check (validate) @ mnemonic-nvml-vmem-service ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.891 s
[INFO] Finished at: 2020-07-18T13:11:39-07:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.0.0:check (validate) on project mnemonic-nvml-vmem-service: Failed during checkstyle configuration: cannot initialize module SuppressionFilter - Unable to find: suppressions.xml -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

```

--- a/bin/mnemonic-checkstyle.xml
+++ b/bin/mnemonic-checkstyle.xml
@@ -164,7 +164,4 @@
 
     <module name="SuppressionCommentFilter"/>
 
-    <module name="SuppressionFilter">
-      <property name="file" value="bin/suppressions.xml"/>
-    </module>
 </module>

--- a/mnemonic-core/pom.xml
+++ b/mnemonic-core/pom.xml
@@ -51,7 +51,7 @@
     <dependency>
       <groupId>com.squareup</groupId>
       <artifactId>javapoet</artifactId>
-    </dependency>    
+    </dependency>
     <!-- logging dependencies -->
     <!-- For core, assume all APIs will be used -->
     <dependency>
@@ -184,7 +184,7 @@
       <build>
         <plugins>
           <plugin>
-            <groupId>org.apache.maven.plugins</groupId> 
+            <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
               <argLine>-Xmx2g -XX:MaxPermSize=1g</argLine>
@@ -192,11 +192,10 @@
                 <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
               </suiteXmlFiles>
             </configuration>
-          </plugin> 
+          </plugin>
         </plugins>
       </build>
     </profile>
   </profiles>
 
 </project>
-

--- a/mnemonic-memory-services/mnemonic-nvml-vmem-service/pom.xml
+++ b/mnemonic-memory-services/mnemonic-nvml-vmem-service/pom.xml
@@ -27,7 +27,7 @@
     <artifactId>mnemonic-memory-services</artifactId>
     <version>0.12.0-SNAPSHOT</version>
   </parent>
-  
+
   <artifactId>mnemonic-nvml-vmem-service</artifactId>
   <name>mnemonic-nvml-vmem-service</name>
 
@@ -150,7 +150,7 @@
       <build>
         <plugins>
           <plugin>
-            <groupId>org.apache.maven.plugins</groupId> 
+            <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
               <argLine>-Xmx2g -XX:MaxPermSize=1g</argLine>
@@ -158,7 +158,7 @@
                 <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
               </suiteXmlFiles>
             </configuration>
-          </plugin> 
+          </plugin>
         </plugins>
       </build>
     </profile>

--- a/pom.xml
+++ b/pom.xml
@@ -290,9 +290,11 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-checkstyle-plugin</artifactId>
-          <version>2.17</version>
+          <version>3.0.0</version>
           <configuration>
             <configLocation>bin/mnemonic-checkstyle.xml</configLocation>
+            <suppressionsLocation>bin/suppressions.xml</suppressionsLocation>
+            <suppressionsFileExpression>checkstyle.suppressions.file</suppressionsFileExpression>
             <includeTestSourceDirectory>true</includeTestSourceDirectory>
             <encoding>UTF-8</encoding>
             <failOnViolation>true</failOnViolation>

